classes/Integration/Form/Forminator.php

```
@@ -0,0 +1,184 @@
+
+		return $form_selectlist;
+	}
+
+	/**
+	 * Hooks in a success function specific to this provider for non AJAX submission handling.
+	 *
+	 * @param object $entry    Form entry object.
+	 * @param int    $form_id  Form ID.
+	 * @param array  $field_data_array Field data array.
+	 */
+	public function on_success( $entry, $form_id, $field_data_array ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$entry'. (undefined)

(UnusedFormalParameter)